documentation/src/docs/controller-service.md

```diff
@@ -455,29 +432,33 @@ to listen for cluster change notifications. These notify about the added and rem
 
 One Controller instance assumes leadership amongst all Controller
 instances. This leader Controller instance is responsible for handling
-Segment Store node change notifications. Based on change in topology,
+Segment Store node change notifications. Based on the changes in topology,
 Controller instance periodically rebalances segment containers to
 Segment Store node mapping.
 
 All Controller instances listen for Controller node change
 notifications. Each Controller instance has multiple sub components that
 implement the _failover sweeper_ interface. Presently there are three
-components that implement failover sweeper interface namely,
-`TaskSweeper`, `EventProcessors` and `TransactionSweeper`. Whenever a
-Controller instance is identified to have been removed from the cluster,
-the cluster listener invokes all registered failover sweepers to
+components that implement _failover_ sweeper interface namely:
+
+- `TaskSweeper`
+- `EventProcessors`
+- `TransactionSweeper`
+
+Whenever a Controller instance is identified to have been removed from the cluster,
+the cluster listener invokes all registered _failover sweepers_ to
 optimistically try to sweep all the orphaned work previously owned by
 the failed Controller host.
 
 ## Host Store
 
-Host store interface is used to store _Segment Container_ to _Segment Store_
+Host store interface is used to store _Segment Container_ to_Segment Store
```

Comment:
An interface does not store anything, the implementation of the interface does. Also, there is a formatting issue in this sentence: `to_Segment Store`

Reply:
Accepted suggestion